### ISSUE TITLE
Suggestion with collapse and minSize

### ISSUE THREAD

**Issue by DemersM** [nathancahill/split#188]
In my point of view, collapse should resize to 0 even if the minSize is superior. The minSize should affect the size when dragging/resizing only.

Then we could use the collapse function to hide the pane while assuring that it wont be smaller than the minSize when visible.

This will be closer to the definition of the word collapse.

**Comment by looeee** [comment]
> This will be closer to the definition of the word collapse.

Not really. If you collapse a real world object, it doesn't shrink to an infinitesimal point, it just goes into its "smallest state". To me, collapsing to `minSize` seems intuitive and correct. If I want the pane to collapse to zero then I'll set 0 as the min size. 

I suspect this is one of those situations where either version of the function will be correct for some use cases and incorrect for others.

**Comment by DaleyKD** [comment]
I agree with the OP. I'm sure there are 20 who will disagree with me. :)

I need to find a way to use Split.js for it's resizing of panes but also to use it like an accordion (or Bootstrap collapsible, whatever).

**Comment by sprint-rest** [comment]
+1 on this enhancement
